submissions/dikun/html-css-popup/index.html

```diff
@@ -0,0 +1,146 @@
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/photos.png" alt="">
+                                    <p>Photos</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/apps.png" alt="">
+                                    <p>Apps</p>
+                                </a>
+                            </li>
+                            <li class="popup__item">
+                                <a href="#" class="popup__link">
+                                    <img class="popup__img" src="icons/documents.png" alt="Documents">
+                                    <p>Documents</p>
+                                </a>
+                            </li>
+                        </ul>
+                        <input type="checkbox" id="check-more" class="input__popup-more">
+                            <label for="check-more" class="label__popup-more">MORE</label>                            
+                            <div class="popup__content-more">
+                                <ul class="popup__lists popup__lists-border">
+                                    <li class="popup__item">
+                                        <a href="#" class="popup__link">
+                                            <img class="popup__img" src="icons/chat.png" alt="">
+                                            <p>Chat</p>
+                                        </a>
+                                    </li>
+                                    <li class="popup__item">
+                                        <a href="#" class="popup__link">
+                                            <img class="popup__img" src="icons/classroom.png" alt="">
+                                            <p>Classroom</p>
+                                        </a>
+                                    </li>
+                                    <li class="popup__item">
+                                        <a href="#" class="popup__link">
+                                            <img class="popup__img" src="icons/blogs.png" alt="">
+                                            <p>Blogs</p>
+                                        </a>
+                                    </li>                                     
+                                </ul>
+                                <input type="checkbox" id="check-even-more" class="input__popup-even">
+                                    <label for="check-even-more" class="label__popup-even">
+                                    <a href="https://github.com/kottans/frontend/blob/2022_UA/tasks/html-css-popup.md" target="_blank">Even more</a>
```

Comment:
Avoid including one interactive element inside another.
When user get focused on button "Even more" and presses key Tab focus must go to the next interactive element, in your case it goes to text. Avoid this situation in the future.